package.json

```diff
@@ -37,7 +37,7 @@
     "test:dependency": "./scripts/dependency-test.sh",
     "test:feature-targeting": "node test/scss/verify-feature-targeting.js",
     "test:site": "npm run clean:site && ./scripts/site-generator-test.sh",
-    "test:unit": "npm run test:jasmineunit; npm run test:mochaunit",
+    "test:unit": "npm run test:jasmineunit; jasmine_exit_code=$?; npm run test:mochaunit && exit \"$jasmine_exit_code\"",
```

Comment:
What will happen if the Mocha tests fail? Will it exit with code 1?

Reply:
Yep, since exit "$jasmine_exit_code" will only execute if `npm run test:mochaunit` succeeds.

Reply:
Cool cool, thanks!